docs/_packages/oauth.md

```diff
@@ -131,13 +131,17 @@ If you decide you need custom success or failure behaviors (ex: wanting to show
 
 ```javascript
 const callbackOptions = {
-  success: (installation, installOptions, req, res) => {
+  success: (installation, metadata, req, res) => {
     // Do custom success logic here
-    res.send('successful!');
+    // tip: you can add javascript and css in the htmlResponse using the <script> and <style> tags
+    const htmlResponse = `<html><body>Success!</body></html>`
+    res.writeHead(200, { 'Content-Type': 'text/html' });
```

Comment:
In general, I recommend having charset here for i18n safety. I know this works without it but we can have it for developers that build their own pages containing multi byte characters.
```suggestion
    res.writeHead(200, { 'Content-Type': 'text/html; charset=utf-8' });
```